src/mame/mame.lst

```diff
@@ -18248,6 +18248,9 @@ topgunnr
 victorba
 victory
 
+@source:f256/f256.cpp
+f256k
+
```

Comment:
You could probably throw this in src/mame/homebrew, but even if you don’t, shouldn’t the project be foenixretro or something?  We avoid single-machine projects.